### ISSUE TITLE
[cetic/nifi] Customize config files such logback.xml

### ISSUE THREAD

**Issue by novakov-alexey** [cetic/helm-nifi#68]
I want to customize logback.xml to add my log appenders

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I want to set a custom ConfigMap name with new logback.xml which needs to be used by Helm install, instead of default file from configs folder.

**Describe alternatives you've considered**
Now I have to fork Helm charts to change logback.xml

**Additional context**

**Comment by novakov-alexey** [comment]
@alexnuttinck do you have any idea how this could be implemented?

**Comment by alexnuttinck** [comment]
https://github.com/cetic/helm-nifi/blob/master/values.yaml#L42 by configmaps in the values.yaml file?

**Comment by novakov-alexey** [comment]
I would need my configmap to override existing configmap, which is:

```
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ template "apache-nifi.fullname" . }}-config
  labels:
    app: {{ include "apache-nifi.name" . | quote }}
    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
    release: {{ .Release.Name | quote }}
    heritage: {{ .Release.Service | quote }}
data:
{{ (tpl (.Files.Glob "configs/*").AsConfig . ) | indent 2 }}
```

Then in Statefulset.yaml, I assume we need to check some new flag if logback.xml should be taken from default file or from user's ConfigMap. This place:

```- name: "logback-xml"
        configMap:
          name: {{ template "apache-nifi.fullname" . }}-config
          items:
            - key: "logback.xml"
              path: "logback.xml"
```

Just using this https://github.com/cetic/helm-nifi/blob/master/values.yaml#L42 won't help I guess.

**Comment by alexnuttinck** [comment]
@novakov-alexey fixed in 0.6.0.
